src/openfermion/ops/operators/symbolic_operator_test.py

```diff
@@ -9,17 +9,20 @@
 #   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
+
 """Tests  symbolic_operator.py."""
+
 import copy
+import numpy
+import sympy
 import unittest
 import warnings
 
-import numpy
-import sympy
 from openfermion.config import EQ_TOLERANCE
 from openfermion.testing.testing_utils import EqualsTester
```

Comment:
The imports are usually grouped in 3 alphabetically sorted blocks: (1) builtin imports (2) third_party imports (3) current package imports.

Can you please put back the numpy, sympy imports _after_ builtins?